server/applicationPattern/applicationPattern/onfModel/services/ForwardingConstructAutomationServices.js

```diff
@@ -164,7 +163,7 @@ function automateForwardingsWithoutInputAsync(forwardingName, attributeList, con
                 let _isForwardingConstructIsProcessSnippet = isForwardingConstructIsProcessSnippet(
                     forwardingConstruct);
                 if (_isForwardingConstructIsProcessSnippet) {
-                    automateProcessSnippetAsync(forwardingConstruct,
+                    await automateProcessSnippetAsync(forwardingConstruct,
```

Comment:
Is there any particular reason for making include await here. 
Kindly please let provide your views.

Reply:
Only that the tests will not pass. It is also part of Aman's PR #813 . It has no effect on the callbacks being async, as the final call to eventDispatcher is not awaited.